game/scenes.py

```diff
@@ -46,6 +46,7 @@
 from .blocks import *
 from .utilities import *
 from .graphics import BlockGroup
+from .generate_world import *
```

Comment:
This module does not exist yet.

Reply:
Oops. Yeah -- I tried to implement a "generate world" that was separate from the scenes.py so that students can write their own proc. gen. algorithms easily!

Reply:
It didn't work -- but I forgot to remove that.